zaza/openstack/charm_tests/vault/tests.py

```diff
@@ -257,6 +257,60 @@ def test_zzz_pause_resume(self):
         lead_client = vault_utils.extract_lead_unit_client(self.clients)
         self.assertTrue(lead_client.hvac_client.seal_status['sealed'])
 
+    def test_vault_reload(self):
+        """Run reload tests.
+
+        Reload service and check services were restarted
+        by doing simple change in the running config by API.
+        Then confirm that service is not sealed
+        """
+        vault_actions = zaza.model.get_actions(
+            'vault')
+        if 'reload' not in vault_actions:
+            raise unittest.SkipTest("The version of charm-vault tested does "
+                                    "not have reload action")
+
+        lead_client = vault_utils.extract_lead_unit_client(self.clients)
+        running_config = vault_utils.get_running_config(lead_client)
+        value_to_set = not running_config['data']['disable_mlock']
+
+        zaza.model.set_application_config(
+            'vault',
+            {'disable-mlock': str(value_to_set)})
+
+        logging.info("Testing reload")
+        zaza.model.run_action_on_leader(
+            'vault',
+            'reload',
+            action_params={})
+
+        self.assertEqual(
+            value_to_set,
+            vault_utils.get_running_config(lead_client)[
+                'data']['disable_mlock'])
+        self.assertFalse(lead_client.hvac_client.seal_status['sealed'])
+
+    def test_vault_restart(self):
+        """Run pause and resume tests.
+
+        Restart service and check services are started.
+        """
+        # Restarting vault process will set it as sealed so it's
```

Comment:
Rather than hoping that this test is run last, it would be better to clean up after itself by unsealing Vault at the end of the test

Reply:
Thanks for pointing that out. That was stuping copy paste from pause resume. Actually the vault gets unselead after every test, what you spotted, in the BaseVaultTest class by tearDown(). 
So I will just remove these misleading comments and we are good.